components/image/slider/src/index.scss

```diff
@@ -1,9 +1,54 @@
-@import '~@schibstedspain/sui-theme/lib/settings-compat-v7/index';
 @import '~@schibstedspain/sui-theme/lib/index';
 @import '~react-slidy/lib/index';
 
+$m-image-slider-counter: $m-m !default;
+$c-image-slider-icon: $c-white !default;
+$sz-image-slider-icon: $sz-icon-s !default;
+$miw-image-slider-counter-text: 34px !default;
+$h-image-slider: 24px !default;
+$fz-image-slider: 12px !default;
+$bgc-image-slider: rgba($c-black, .75) !default;
+$c-image-slider: $c-white !default;
+
 .sui-ImageSlider {
+  position: relative;
+
   &-image {
     max-width: 100%;
   }
+
+  &-counter {
+    align-items: center;
+    background-color: $bgc-image-slider;
+    border-radius: $bdrs-m;
+    color: $c-image-slider;
+    display: flex;
+    font-size: $fz-image-slider;
+    height: $h-image-slider;
+    padding-left: $p-m;
+    padding-right: $p-m;
+    position: absolute;
+
+    &--bottom-right {
+      bottom: $m-image-slider-counter;
+      right: $m-image-slider-counter;
+    }
+
+    &--bottom-left {
+      bottom: $m-image-slider-counter;
+      left: $m-image-slider-counter;
+    }
+
+    &-icon {
+      fill: $c-image-slider-icon !important;
+      height: $sz-image-slider-icon;
+      width: $sz-image-slider-icon;
+      margin-right: $m-s;
+    }
+
+    &-text {
+      min-width: $miw-image-slider-counter-text;
+      text-align: right;
+    }
+  }
```

Comment:
As far as i know, classnames should be defined like so:

```scss
sui-ImageSlider-counterText
sui-ImageSlider-counterIcon
```

btw, not really sure about how should we define the modifiers.
there is my proposition:

```scss
sui-ImageSlider-counter--bottomLeft
sui-ImageSlider-counter--bottomRight
```

Reply:
Absolutely. My mistake 👍